lms/static/js/discussions_management/views/discussions.js

```diff
@@ -108,18 +108,13 @@
                 showCohortSchemeControl: function(show) {
                     if (!show) {
                         $('.division-scheme-item.cohort').addClass(HIDDEN_CLASS);
-                        this.updateSchemeSelectionLayout(2);
-                    } else {
-                        $('.division-scheme-item.cohort').removeClass(HIDDEN_CLASS);
-                        this.updateSchemeSelectionLayout(3);
-                    }
-                },
-
-                updateSchemeSelectionLayout: function(columns) {
-                    if (columns === 2) {
+                        // Since we are removing the cohort scheme, we can have at most 2 columns.
                         $('.division-scheme-item').removeClass(THREE_COLUMN_CLASS).addClass(TWO_COLUMN_CLASS);
                     } else {
-                        $('.division-scheme-item').removeClass(TWO_COLUMN_CLASS).addClass(THREE_COLUMN_CLASS);
+                        $('.division-scheme-item.cohort').removeClass(HIDDEN_CLASS);
+                        if (this.isSchemeAvailable(ENROLLMENT_TRACK)) {
```

Comment:
It turns out this logic isn't complete. We also need https://github.com/edx/edx-platform/pull/15254/files#r120504369.